README.md

```diff
@@ -5,8 +5,8 @@ The GSLab Template is a minimal working demonstration of the tools and organizat
 ## Prerequisites
 
 You'll need the following to run the template. [Homebrew](https://brew.sh/) for Mac and [Linuxbrew](http://linuxbrew.sh/) for Linux make this easier.
-* Windows `cmd.exe`, Mac OS X `bash`, or Linux `bash`. 
-* [Python 2.7.X](https://wiki.python.org/moin/BeginnersGuide/Download) for [Windows](https://docs.python.org/2/using/windows.html), [Mac](https://docs.python.org/2/using/mac.html) or [Linux](https://docs.python.org/2/using/unix.html).
+* Windows `cmd.exe`, Mac OS X `bash`, or Linux `bash`.
+* [Python 2.7.X](https://wiki.python.org/moin/BeginnersGuide/Download) and [pip](https://pip.pypa.io/en/stable/installing/) for [Windows](https://docs.python.org/2/using/windows.html), [Mac](https://docs.python.org/2/using/mac.html) or [Linux](https://docs.python.org/2/using/unix.html).
```

Comment:
We already have a section on this [here](https://github.com/gslab-econ/template/blob/2c30ad8bb276736db4d645000f715810767e2a33/README.md#where-should-i-track-the-prerequisites-for-my-repository). Do you want it moved somewhere else?

Reply:
I suggest two changes:

1) Make explicit that the required packages are stored in `config_<soft>.<ext>` scripts, and make explicit where they are store with a flashy comment to draw attention. I would like to reduce the amount of work needed for someone to figure out where these packages are listed as much as possible.

2) There should be a note in the setup (i.e. where I commented) about packages since these are prerequisites (even though we're installing the prerequisites for them). No one in the history of using softwares has ever read through the FAQs before trying to run it for the first time and I think this is a potential failure point.

Reply:
See https://github.com/gslab-econ/template/pull/98/commits/7b0d1555534ae58972d55c1a2df78178e5ddeedc. LMK if you want more.